eclass/texlive-module.eclass

```diff
@@ -233,16 +233,16 @@ texlive-module_make_language_lua_lines() {
 	local lefthyphenmin righthyphenmin synonyms name file file_patterns file_exceptions luaspecial
 	local dest="${S}/language.${PN}.dat.lua"
 	eval $@
-	[ -z "$lefthyphenmin"  ] && lefthyphenmin="2"
-	[ -z "$righthyphenmin" ] && righthyphenmin="3"
+	[[ -z ${lefthyphenmin}  ]] && lefthyphenmin="2"
+	[[ -z ${righthyphenmin} ]] && righthyphenmin="3"
 	einfo "Generating language.dat.lua entry for $@"
 	printf "\t['%s'] = {\n" "$name"                                                                 >> "$dest"
 	printf "\t\tloader = '%s',\n" "$file"                                                           >> "$dest"
 	printf "\t\tlefthyphenmin = %s,\n\t\trighthyphenmin = %s,\n" "$lefthyphenmin" "$righthyphenmin" >> "$dest"
 	printf "\t\tsynonyms = {%s },\n" "$(texlive-module_synonyms_to_language_lua_line "$synonyms")"  >> "$dest"
-	[ -n "$file_patterns"   ] && printf "\t\tpatterns = '%s',\n" "$file_patterns"                   >> "$dest"
-	[ -n "$file_exceptions" ] && printf "\t\thyphenation = '%s',\n"	"$file_exceptions"              >> "$dest"
-	[ -n "$luaspecial"      ] && printf "\t\tspecial = '%s',\n" "$luaspecial"                       >> "$dest"
+	[[ -n ${file_patterns}   ]] && printf "\t\tpatterns = '%s',\n" "$file_patterns"                   >> "$dest"
+	[[ -n ${file_exceptions} ]] && printf "\t\thyphenation = '%s',\n"	"$file_exceptions"              >> "$dest"
+	[[ -n ${luaspecial}      ]] && printf "\t\tspecial = '%s',\n" "$luaspecial"                       >> "$dest"
```

Comment:
1. all of this stuff needs `|| die`
2. if you're aligning the `[[ ]]` (which I personally disagree with, but don't care), at least align the `>> $dest` part too